components/usage/pkg/db/billed_session.go

```diff
@@ -27,3 +29,23 @@ type BilledSession struct {
 func (d *BilledSession) TableName() string {
 	return "d_b_billed_session"
 }
+
+func SetBilled(ctx context.Context, conn *gorm.DB, instanceID uuid.UUID, instanceCreationTime time.Time, system string) error {
+	database := conn.WithContext(ctx)
+
+	billedSession := BilledSession{InstanceID: instanceID, From: NewVarcharTime(instanceCreationTime), System: system}
+	db := database.Create(&billedSession)
+
+	return db.Error
+}
+
+func GetBilled(ctx context.Context, conn *gorm.DB, instanceID uuid.UUID) ([]BilledSession, error) {
```

Comment:
This function isn't used anywhere. Should it be part of this PR?

Reply:
True. I wrote this placeholder method due to [this comment](https://gitpod.slack.com/archives/C02EN94AEPL/p1659600593291589?thread_ts=1659591950.914839&cid=C02EN94AEPL), but you're right it's not necessary. Happy to remove it.